consul-lambda/go.mod

```diff
@@ -1,22 +1,22 @@
 module github.com/hashicorp/terraform-aws-consul-lambda/consul-lambda
 
-go 1.18
+go 1.19
 
 require (
 	github.com/aws/aws-lambda-go v1.31.1
 	github.com/aws/aws-sdk-go-v2 v1.16.3
 	github.com/aws/aws-sdk-go-v2/config v1.15.5
 	github.com/aws/aws-sdk-go-v2/service/lambda v1.22.1
 	github.com/aws/aws-sdk-go-v2/service/ssm v1.27.0
-	github.com/google/go-cmp v0.5.7
-	github.com/hashicorp/consul v1.13.1
-	github.com/hashicorp/consul/api v1.14.0
-	github.com/hashicorp/consul/sdk v0.11.0
-	github.com/hashicorp/go-hclog v0.14.1
+	github.com/google/go-cmp v0.5.8
+	github.com/hashicorp/consul v1.11.0-beta1
+	github.com/hashicorp/consul/api v1.10.1-0.20230120175926-25396d81c9ee
```

Comment:
we can address this after the `1.15` release.

Reply:
yeah, we'll have to change this, I'll mention this in the follow up PR I create